DI-811 Extract functionality

gaborkorodi-hms opened the pull request:
Removing the leading/trailing spaces.

Commenting out the deletion of the original file.

Commit:
DI-811 Extract functionality

Removing the leading/trailing spaces.

Commenting out the deletion of the original file.

namdets commented:
Approving this to encourage progress, but the cleanup of the files should be reinstituted before this makes it to production to ensure Docker environment compatibility.

Commit:
DI-811 Extract functionality

Improve the formatting of the output file. Remove padding spaces and formalize the wrapping of character data within double quotes, as done by standard functions.

ghost commented:
I have made some additional changes (for formatting, NOT to the logic). Also, I've put back the removal of the intermediary file.

We still cannot find anybody to approve this?
